src/Microsoft.Extensions.DependencyModel/CompilationLibrary.cs

```diff
@@ -51,31 +51,31 @@ public class CompilationLibrary : Library
 
         public IEnumerable<string> ResolveReferencePaths()
         {
-            return ResolveReferencePaths(DefaultResolver);
+            var assemblies = new List<string>();
+
+            return ResolveReferencePaths(DefaultResolver, assemblies);
         }
 
-        public IEnumerable<string> ResolveReferencePaths(params ICompilationAssemblyResolver[] additionalResolvers)
+        public IEnumerable<string> ResolveReferencePaths(params ICompilationAssemblyResolver[] customResolvers)
```

Comment:
I'm not sure what the advantage of that approach is.  What problems can you solve with that API that you can't with this one?